src/components/molecules/DropdownFiltered/__snapshots__/DropdownFiltered.test.tsx.snap

```diff
@@ -63,15 +63,20 @@ exports[`<DropdownFiltered /> renders the Dropdown with options with no a11y vio
   transition-property: border,box-shadow;
   -webkit-transition: 0.2s ease-in-out;
   transition: 0.2s ease-in-out;
+  font-family: "Open Sans",Roboto,Helvetica,Arial,sans-serif;
```

Comment:
Is this deliberate?

Reply:
same here as DropdownFiltered is using InputText which is now using the font from the theme